### PR TITLE
LPS-145232 Replace ID with namespace and remove boilerplate

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/asset/asset-info-impl/src/main/java/com/liferay/asset/info/internal/item/AssetEntryInfoItemFields.java [hunk 1]
```diff
@@ -16,143 +16,117 @@
...
-	).build();
+	public static final InfoField userProfileImageInfoField =
+		BuilderStaticHolder._builder.infoFieldType(
+			TextInfoFieldType.INSTANCE
+		).name(
+			"userProfileImage"
+		).labelInfoLocalizedValue(
+			InfoLocalizedValue.localize(
+				AssetEntryInfoItemFields.class, "user-profile-image")
+		).build();
```
Comment (adolfopa) [inline modules/apps/asset/asset-info-impl/src/main/java/com/liferay/asset/info/internal/item/AssetEntryInfoItemFields.java:114]: I made a mistake here; it should be `ImageInfoFieldType`. I'll fix this before sending to Brian.

**Comment by victorg1991** [comment]
@adolfopa  LGTM :)  I have asked @jkappler to take an additional look since he knows way more about the info framework than me :p

**Comment by jkappler** [comment]
@adolfopa, @victorg1991, don't have anything to add, looks good ❤️

**Pull request closed without merging**